src/pages/workspace/companyCards/assignCard/AssigneeStep.tsx

```diff
@@ -27,19 +27,23 @@ import type * as OnyxTypes from '@src/types/onyx';
 const MINIMUM_MEMBER_TO_SHOW_SEARCH = 8;
 
 type AssigneeStepProps = {
-    // The policy that the card will be issued under
+    /** The policy that the card will be issued under */
     policy: OnyxEntry<OnyxTypes.Policy>;
+
+    /** Selected feed */
+    feed: OnyxTypes.CompanyCardFeed;
 };
 
-function AssigneeStep({policy}: AssigneeStepProps) {
+function AssigneeStep({policy, feed}: AssigneeStepProps) {
     const {translate} = useLocalize();
     const styles = useThemeStyles();
     const {isOffline} = useNetwork();
     const [assignCard] = useOnyx(ONYXKEYS.ASSIGN_CARD);
     const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policy?.id ?? '-1');
 
-    const [list] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${assignCard?.data?.bankName ?? ''}`);
-    const filteredCardList = CardUtils.getFilteredCardList(list);
+    const [list] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${feed}`);
+    const [cardFeeds] = useOnyx(`${ONYXKEYS.COLLECTION.SHARED_NVP_PRIVATE_DOMAIN_MEMBER}${workspaceAccountID}`);
+    const filteredCardList = CardUtils.getFilteredCardList(list, cardFeeds?.settings?.oAuthAccountDetails?.[feed]);
```

Comment:
@koko57 I think we should pass workspaceAccountID and feed to `getFilteredCardList` function and we will get Onyx data on it. And make workspaceAccountID and feed param as required

Reply:
Discussed in the [slack](https://callstack-hq.slack.com/archives/C06ML6X0W9L/p1732717522601619?thread_ts=1732550511.971919&cid=C06ML6X0W9L).
Overall with this approach we can met unexpected issues, cause `CardUtils.getFilteredCardList` function wan't be re-triggered inside the component when card list is updated